test/activations.jl

```diff
@@ -7,9 +7,27 @@
 end
 
 @testset "forward diff" begin
-    f(x) = logσ.(x)
+    f = (x) -> logσ.(x)
     ds = Dual.(rand(5),1)
     @test f(ds) ≈ collect(f(CuArray(ds)))
+    f = (x) -> tanh.(x)
+    ds = Dual.(rand(5),1)
+    @test f(ds) ≈ collect(f(CuArray(ds)))
+    f = (x) -> σ.(x)
+    ds = Dual.(rand(5),1)
+    @test f(ds) ≈ collect(f(CuArray(ds)))
+    f = (x) -> elu.(x)
+    ds = Dual.(rand(5),1)
+    @test f(ds) ≈ collect(f(CuArray(ds)))
+    f = (x) -> relu.(x)
+    ds = Dual.(rand(5),1)
+    @test f(ds) ≈ collect(f(CuArray(ds)))
+end
+
+@testset "complex" begin
+    f = (x) -> tanh.(x)
```

Comment:
```suggestion
    f(x) = tanh.(x)
```

Reply:
A comment mentioning why this test exists would be good too. e.g. broadcasting over a complex CuArray should work without NNlibCUDA, but this ensures that NNlibCUDA doesn't accidentally make this take a fast path that doesn't support complex numbers.

Reply:
I've added the comment, thanks!